### DISCUSSION TITLE
Can multiple passwords coexist?😂

### DISCUSSION THREAD

**Discussion by maxbad** [utelle/SQLite3MultipleCiphers#28] [Q&A] [answered]
Login with different passwords can successfully see the data😁🤞

**Comment by utelle** [comment, marked as answer]
If you are talking about the [SQLite User Authentication](https://www.sqlite.org/src/doc/trunk/ext/userauth/user-auth.txt) extension, then the answer to your question is **yes**. However, please note that this extension is about _user authentication_, **not** _database encryption_. For its use please consult [its documentation](https://www.sqlite.org/src/doc/trunk/ext/userauth/user-auth.txt).

_SQLite3 Multiple Ciphers_ includes the _SQLite User Authentication_ extension as well as several other extensions, which are enabled by default in the build files coming with _SQLite3 Multiple Ciphers_, but can be disabled by removing the corresponding preprocessor symbol.

The main feature of _SQLite3 Multiple Ciphers_ is the support of several database encryption schemes. If database encryption is activated, then you have a _single passphrase_ protecting the database. All users knowing this single passphrase can access the database.

Of course, it is possible to combine _database encryption_ and _user authentication_. The process how to open a database in this case is described in the [SQLite User Authentication](https://www.sqlite.org/src/doc/trunk/ext/userauth/user-auth.txt) documentation.

**Comment by utelle** [comment]
> Closing ... because I think this is a non-issue. It's simply a matter of correctly using the various features offered by _SQLite3 Multiple Ciphers_.

I converted the issue into a discussion. Maybe others find the given information useful.
